base_tier_validation/models/tier_review.py

```diff
@@ -173,7 +172,9 @@ def _get_reminder_activity_type(self):
 
     def _notify_review_reminder_body(self):
         delay = (fields.Datetime.now() - self.create_date).days
-        return _("A review has been requested %s days ago.") % (delay)
+        return self.env._(
+            "A review has been requested %(delay)s days ago.", delay=delay
```

Comment:
Same as previous comment.
Since there's only 1 param, IMO there's no benefit in using a kword argument and it's better to keep the original translate term